src/app/components/connect-applications-wizard/connect-applications-wizard.less

```diff
@@ -0,0 +1,69 @@
+#connect-apps-panel {
+  min-height: 85%;
+  max-height: 85%;
```

Comment:
I have read that using max-height and min-height, I gain more control. In fact, when I switch this to height:85%, the overflow hidden scrollable stop working and therefore the elements come out the box.
What should I do?